src/main/charts/confluence/templates/config-jvm.yaml

```diff
@@ -16,6 +16,9 @@ data:
     {{ . }}
     {{- end }}
     -XX:ActiveProcessorCount={{ include "flooredCPU" .Values.confluence.resources.container.requests.cpu }}
+    {{ if .Values.confluence.s3AttachmentsStorage.eksIrsa.roleArn }}
+    -Daws.webIdentityTokenFile=/var/run/secrets/eks.amazonaws.com/serviceaccount/token
```

Comment:
Since by default `AWS_WEB_IDENTITY_TOKEN_FILE ` which EKS sets will be unset by the [image entrypoint](https://bitbucket.org/atlassian-docker/docker-atlassian-confluence-server/src/4ee36a2c95622f1312ec2d661b1d013132a0300b/entrypoint.py#lines-21),  adding its equivalent as a Java property.